usaspending_api/reporting/v2/views/agencies/publish_dates.py

```diff
@@ -39,8 +40,14 @@ def get_agency_data(self):
         if self.filter is not None:
             agency_filters.append(Q(name__icontains=self.filter) | Q(abbreviation__icontains=self.filter))
         results = (
-            ToptierAgency.objects.account_agencies()
-            .filter(*agency_filters)
+            ToptierAgency.objects.annotate(
```

Comment:
Could you implement something similar to `account_agencies` so that Code Climate will stop complaining? 

One thought is to change `ToptierAgencyManager.account_agencies()` to accept a parameter that specifies the agency type. Have that parameter default to `funding` to avoid breaking other places that use that method.

Reply:
yea , that's a good idea